core/templates/pages/contributor-dashboard-page/opportunities-list-item/opportunities-list-item.component.html

```diff
@@ -7,6 +7,11 @@
            class="progress">
         <div class="progress-bar bg-success"
              role="progressbar"
+             [attr.aria-label]="progressBarLabel"
```

Comment:
https://angular.io/guide/attribute-binding   --->Angular documentation thats reason i have used it

attr.aria-label used because in Angular to bind an aria-label attribute dynamically to an element using attribute binding and  Angular will update the value of the attribute whenever the value of the property changes if we use only 
aria-label we should manually  set the label based on some value in the component,

Reply:
Gotcha, thanks!